### ISSUE TITLE
Few jshint errors

### ISSUE THREAD

**Issue by arkadiyk** [runspired/ember-mobiletouch#50]
```
modules/ember-mobiletouch/components/action-area.js: line 79, col 12, Don't make functions within a loop.

1 error

modules/ember-mobiletouch/components/draggable-item.js: line 6, col 3, 'on' is defined but never used.
modules/ember-mobiletouch/components/draggable-item.js: line 9, col 8, 'get' is defined but never used.
modules/ember-mobiletouch/components/draggable-item.js: line 176, col 22, 'e' is defined but never used.
modules/ember-mobiletouch/components/draggable-item.js: line 182, col 19, 'e' is defined but never used.

4 errors

modules/ember-mobiletouch/components/slide-toggle.js: line 64, col 17, 'value' is already defined.
modules/ember-mobiletouch/components/slide-toggle.js: line 8, col 3, 'observer' is defined but never used.

2 errors

modules/ember-mobiletouch/lib/ember-imports.js: line 25, col 18, 'get' is not defined.

1 error

modules/ember-mobiletouch/overrides/event-dispatcher.js: line 5, col 8, 'isGesture' is defined but never used.

1 error

modules/ember-mobiletouch/utils/make-helper-with-args.js: line 36, col 4, Missing semicolon.

1 error
```

**Comment by runspired** [comment]
@arkadiyk these are all part of the 2.0 beta and will get cleaned up before an official 2.0 lands.

**Comment by devotox** [comment]
could they not be commented out and some are just plain jshint errors like 


modules/ember-mobiletouch/components/slide-toggle.js: line 64, col 17, 'value' is already defined.


modules/ember-mobiletouch/utils/make-helper-with-args.js: line 36, col 4, Missing semicolon.

modules/ember-mobiletouch/components/action-area.js: line 79, col 12, Don't make functions within a loop.

**Comment by runspired** [comment]
dev is on hold while I figure out how I'm going to support the improved actions that landed in 1.13 

https://github.com/emberjs/rfcs/blob/00ac2685c86f27d41547012903f485a4ef338d27/active/0000-improved-actions.md

But yes, jshint errors will all be cleaned up before 2.0 release :)
